cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/SchemaService.java

```diff
@@ -150,6 +183,9 @@ public Map<Integer, Map<String, Object>> getSchema(
           getSchemaVersions(environmentVal, topicName, protocol, clusterIdentification);
       String schemaCompatibility =
           getSubjectSchemaCompatibility(environmentVal, topicName, protocol, clusterIdentification);
+      if (Objects.equals(schemaCompatibility, SCHEMA_COMPATIBILITY_NOT_SET))
+        schemaCompatibility =
+            getGlobalSchemaCompatibility(environmentVal, protocol, clusterIdentification);
```

Comment:
I think we should always add `{}`. It will save quite some mistakes in the long run.